_data-prepper/pipelines/configuration/processors/parse-json.md

```diff
@@ -1,27 +1,79 @@
 ---
 layout: default
-title: Parse JSON
+title: Parse_JSON
 parent: Processors
 grand_parent: Pipelines
 nav_order: 45
 ---
 
-# Parse JSON
+# Parse_JSON
 
-## Overview
+The `Parse_JSON` processor parses JSON data for an event, including any nested fields. The processor extracts the JSON pointer data and adds the input event to the extracted fields.
 
-The `parse_json` processor parses JSON data for an event, including any nested fields. The following table describes several optional parameters you can configure in the `parse_json` processor. 
 
-Option | Required | Type | Description
-:--- | :--- | :--- | :---
-source | No | String | The field in the `Event` that will be parsed. Default value is `message`.
-destination | No | String | The destination field of the parsed JSON. Defaults to the root of the `Event`. Cannot be `""`, `/`, or any whitespace-only `String` because these are not valid `Event` fields.
-pointer | No | String | A JSON Pointer to the field to be parsed. There is no `pointer` by default, meaning the entire `source` is parsed. The `pointer` can access JSON Array indices as well. If the JSON Pointer is invalid then the entire `source` data is parsed into the outgoing `Event`. If the pointed-to key already exists in the `Event` and the `destination` is the root, then the pointer uses the entire path of the key.
+## Configuration
 
-<!---## Configuration
+You can configure the `Parse_JSON` processor with the following options.
 
-Content will be added to this section.
+| Option | Required | Type | Description |
+| :--- | :--- | :--- | :--- | 
+| `source` | No | String | The field in the `Event` that will be parsed. Default value is `message`. |
+| `destination` | No | String | The destination field of the parsed JSON. Defaults to the root of the `Event`. Cannot be `""`, `/`, or any whitespace-only `String` because these are not valid `Event` fields. |
```

Comment:
```suggestion
| `destination` | No | String | The destination field of the parsed JSON. Defaults to the root of the `event`. Cannot be `""`, `/`, or any whitespace-only string because these are not valid `event` fields. |
```